src/hooks/useCommands.js

```diff
@@ -0,0 +1,21 @@
+import Fuse from 'fuse.js'
+
+const useCommands = (commands, precision = 0.4) => {
```

Comment:
The precision parameter should be customisable by callees of <Vocal />. 0.4 seems very low to me.

Reply:
I thought of making the value customisable but since this score is not very explicit it would more confusing than helping. I may do it in the future if I find the right way to explain what does it mean.